endpoints/_includes/functions.php

```diff
@@ -746,6 +748,30 @@ function helplineSearch($latitude, $longitude)
     return json_decode(get($search_url));
 }
 
+function getBMLTserver($latitude, $longitude, $search_state)
```

Comment:
There are a lot of formatting issues going on here.  There should be spaces between equal signs, periods for string concatenation (`=`, `.`).  curly brackets are not lining up like the rest of the code base.